### PR TITLE
[WP] Base ansible-galaxy-os on ubuntu-xenial (16.04)

### PR DESCRIPTION
**Pull request by drosofff**
This PR

- switches to the use of a VM ubuntu xenial for travis environment (as opposed to a Docker container 14.04)
- offers 2 Dockerfiles for ubuntu, 14.04 and 16.04 respectively. Since ansible-galaxy-os is used as a first docker layer for a number of Docker builds, this offers more flexibility for these builds
- use the nginx ppa pinning from galaxyproject for all ubuntu versions
- start to fix ansible synthax for compatibility with ansible 2.7

Let me know thoughts, and merry Christmas !

### CODE REVIEW

**Comment by drosofff** [comment]
Happy new year everybody !
Poke somebody for the PR

**Comment by drosofff** [comment]
@afgane @mvdbeek any chance to get your thoughts on this ?
Cheers

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,16 @@
 language: python
 python: 2.7
+dist: trusty
+sudo: required
```
Comment (mvdbeek) [inline .travis.yml]: This is the new default, so shouldn't be necessary.

File: tasks/ubuntu/packages.yml [hunk 2]
```diff
@@ -28,83 +28,41 @@
...
-    - samtools
-    - slurm-drmaa-dev
-    - supervisor
-    - swig
-    - sysstat
-    - unzip
-    - wget
-    - zlib1g-dev
+  apt:
+    pkg: ['ant', 'cmake', 'curl', 'g++', 'gcc', 'gfortran', 'git-core',
```
Comment (mvdbeek) [inline tasks/ubuntu/packages.yml]: Is there a reason to switch the list syntax ?
```
apt:
  pkg:
    - ant
    - cmake
    ...
```
should work too. This looks cleaner to me, and you'd get cleaner diffs when updating.

**Review by mvdbeek — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (drosofff) [reply .travis.yml]: Ah ok. Did not know. Thanks

File: tasks/ubuntu/packages.yml (hunk 2 above)
Comment (drosofff) [reply tasks/ubuntu/packages.yml]: Well, I read the warnings of ansible 2.7 too fast. I also prefer the dashes for list description. I should definitely work

**Comment by afgane** [comment]
I'll jump in after @mvdbeek did all the hard review work and merge ;)

Thanks to both.

**Pull request merged**